src/tests/ut/data_file_manager/data_file_manager_test.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "clinic_facade.hpp"
+#include "gtest/gtest.h"
+
+class DataFileManagerTestFixture : public ::testing::Test
+{
+    void TearDown() override
+    {
+        for (const auto& doctor : Clinic::getDoctors())
+        {
+            Clinic::removeDoctor(doctor);
+        }
+
+        for (const auto& receptionist : Clinic::getReceptionists())
+        {
+            Clinic::removeReceptionist(receptionist);
+        }
+    }
+};
```

Comment:
can include test utils and call universal clinic cleaner

Reply:
done